Explorer/Assets/Scripts/ECS/StreamableLoading/Components/GetTextureIntention.cs

```diff
@@ -0,0 +1,10 @@
+﻿using ECS.StreamableLoading.Components.Common;
+
+namespace ECS.StreamableLoading.Components
+{
+    public struct GetTextureIntention : ILoadingIntention
+    {
+        public CommonLoadingArguments CommonArguments { get; set; }
```

Comment:
Why do we prefer an interface here over a base class?

Reply:
It's a struct - no base classes